evcxr/src/command_context.rs

```diff
@@ -619,6 +626,16 @@ Panic detected. Here's some useful information if you're filing a bug report.
             bail!("Variable does not exist: {}", args)
         }
     }
+
+    fn hover(&mut self, state: &mut ContextState, args: &Option<String>) -> Result<EvalOutputs, Error> {
+        let args = if let Some(x) = args {
+            x.trim()
+        } else {
+            bail!("Input required")
+        };
+        let res = self.eval_context.hover(args, state)?;
+        Ok(EvalOutputs::text_html(res.clone(), res))
```

Comment:
The first argument to `text_html` should ideally be plain text and the second argument HTML. At the moment you're passing markdown for both.

It looks like the two formats that rust-analyzer supports are plain text and markdown. For the first argument, it could be good to pass actual plain text. For the second argument, one possibility would be to use the `markdown` crate to convert the markdown to HTML.

Reply:
I fixed it in this way: 
I get both the plain text and markdown, and then convert the markdown with using a simple markdown html (I am trying to use a more proper markdown crate mdbook(https://github.com/rust-lang/mdBook)). And then parse the two to method `text_html`.